examples/fakeserver/src/main/java/com/stellarstation/api/fakeserver/FakeStellarStationService.java

```diff
@@ -81,11 +87,15 @@
     return new StreamObserver<SatelliteStreamRequest>() {
       @Override
       public void onNext(SatelliteStreamRequest value) {
-        if (!value.getSatelliteId().equals("5")) {
+        if (!value.getSatelliteId().equals(SATELLITE_ID)) {
           throw new StatusRuntimeException(Status.INVALID_ARGUMENT);
         }
-        for (ByteString payload : value.getSendSatelliteCommandsRequest().getCommandList()) {
-          sendTelemetry(payload, responseObserver);
+        if (value.hasTelemetryReceivedAck()) {
+          logger.info("received TelemetryReceivedAck message: \n" + value);
```

Comment:
Could this cause a large amount of logs ?

Reply:
yes it does, I figured this is fake-api-server so should be ok (?). the justification for adding logging is to allow clients to see if they are implementing message acks correctly. let me know if you think we should not log this.